### ISSUE TITLE
Extension Range is not creating the correct number of extensions

### ISSUE THREAD

**Issue by blackc2004** [fusionpbx/fusionpbx#2020]
On master, when creating a new extension if you choose RANGE it's not creating the correct number of extensions.

For instance, I started at 1001 and selected 5, it only created 1001,1002,1003.

**Comment by andycol** [comment]
yes same problem here, i started at 100 and added 25 it only creates 8 extensions

**Comment by bigfishvoice** [comment]
Has this been fixed yet?  Using range also doesn't carry forward the voice mail password and vm to email settings to all of the copied extensions.

**Comment by andycol** [comment]
still not fixed
